addons/overheating/stringtable.xml

```diff
@@ -176,6 +194,12 @@
             <Chinese>利用重裝彈匣來解決卡彈</Chinese>
             <Hungarian>Az újratöltés megszünteti a fegyver elakadását.</Hungarian>
         </Key>
+        <Key ID="STR_ACE_Overheating_unJamOnSwapBarrel_displayName">
+            <English>Unjam on barrel swap</English>
```

Comment:
```suggestion
            <English>Unjam on Barrel Swap</English>
```

Reply:
Most of the other setting names that are not titles use sentence case, such as "Reloading clears a weapon jam", should I title case all of them?

Reply:
Yes, I think that would be good.